src/homework/day8/absclasses/collections/FiguresAsList.java

```diff
@@ -0,0 +1,45 @@
+package homework.day8.absclasses.collections;
+
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+//2.
+//Создать список строк стрингов figures из 5 фигур (Овал, Прямоугольник, Круг, Квадрат, Эллипс) через (Arrays.asList())
+//Проитерировать список через for-each и отпечатать слова в файл figures через тире
+//Посчитать сколько фигур НЕ содержат букву "и" и вывести число в консоль
+//Проитерировать список по индексу и отпечатать слова в консоль через пробел
+//Вставить на 3 позицию Треугольник
+//Проитерировать список через for-each и отпечатать слова в консоль через пробел
+
+public class FiguresAsList {
+    public static void main(String[] args) {
+        List<String> figures = Arrays.asList("Овал", "Прямоугольник", "Круг", "Квадрат", "Эллипс");
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter("file_with_figures", true));
```

Comment:
here I am not sure, should we each time add objects to file, in the task it looks like we should only add figures once, but if code is launched twice or more - file will be updated with new line of objects

Reply:
removed "true" in file writer, now file will be rewritten